cpp/ps0003/ps0003_Solution.cpp

```diff
@@ -1,40 +1,21 @@
-
-  /* help your brother check if the number is even or odd
-        - if the number is even, print "Number is even"
-        - if the number is odd, print "Number is odd"
-
-        - ex:
-          => 909
-          <= Number is odd
-
-        - another ex:
-          => 20000
-          <= Number is odd
-    */
-
-
 #include<iostream>
 
 int main() {
 
-  
-
-
-
+    // prompting user for number 2/2
     int number;
-
     std::cin >> number;
 
+    // checking if the number is odd or even 3/3
     if (number%2==0)
     {
         std::cout<<"Number is even"<< std::endl;
-
     }
     else
     {
         std::cout<<"Number is odd"<< std::endl;
     }

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console